app/static/settings_style.css

```diff
@@ -0,0 +1,143 @@
+/* Settings page */
+
+.sub-title {
+    font-size: 1rem;
+    padding-top: 0.2rem;
+}
+
+.settings {
+  color: #222831;
+  display: flex;
+  font-size: 0.9rem;
+}
+
+#settings-left {
+  padding: 1.5rem 1.5rem;
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(230, 230, 230)
+}
+
+.settings-headline {
+  display: flex;
+  width: 200px;
+  padding-bottom: 1rem;
+}
+
+.settings-layout {
+  display: flex;
+}
+
+.left-options-bar {
+  display: flex;
+}
+
+.settings-main article {
+  margin-bottom: 2rem;
+}
+
+.settings-options {
+  flex: 1
+}
+
+.settings-options ul {
+  list-style-type: none;
+  padding-left: 0;
+  padding-right: 1rem;
+}
+
+.settings-options ul li {
+  margin-bottom: 15px;
+  font-weight: bold;
+}
+
+.settings-options ul li a {
+  color: #222831;
+  transition: all .3s ease-in-out;
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.settings-options ul li a:hover, .settings-options ul li a.active {
+  color: #5786f5;
+}
+
+.tabShow {
+  transition: all .5s ease-in;
+}
+
+.settings-main {
+  flex: 3;
+  padding: 1.5rem 1.5em;
+}
+
+.settings-main h2 {
+  margin-bottom: 1rem;
+}
+
+.settings-main p {
+  margin: 1rem 0rem;
+}
+
+.settings-main .form-select {
+  width: 250px;
+}
+
+#settings_content h2, .settings_content p {
+  display: block;
+  align-self: left;
+}
+
+#settings_content h2 {
+  font-weight: bold;
+}
+
+#select-form-settings {
+  font-size: 0.8rem;
+}
+
+/* For Mobile */
+@media screen and (max-width: 700px) { 
+  .settings {
+    display: block;
+    font-size: 3.5vw;
+  }
+  
+  #settings-left {
+    padding: 3rem 0em 0em 4rem;
+    width: 100%;
+  }
+  
+  .settings-headline {
+    padding-bottom: 20px;
+  }
+  
+  .settings-options {
+    margin-bottom: 30px;
+  }
+
+  .settings-options ul {
+    padding-left: 0;
+  }
+
+  .settings-options ul li {
+    margin-bottom: 5px;
+    font-weight: bold;
+  }
+
+  .settings-main {
+    padding: 4rem 4rem;
+  }
+  .settings-main .form-select {
+    width: 250px;
+  }
+
+  .settings-main h2 {
+    margin-bottom: 1rem;
+  }
+    
+  .settings-main p {
+    margin: 1rem 0px;
```

Comment:
em

Reply:
I don't know if I got it right, but I think I should use em.